sdk/eventhub/Azure.Messaging.EventHubs/src/Primitives/EventProcessor.cs

```diff
@@ -781,6 +781,38 @@ async Task performProcessing()
         ///
         protected abstract Task<IEnumerable<EventProcessorCheckpoint>> ListCheckpointsAsync(CancellationToken cancellationToken);
```

Comment:
Do we want to tag this as `[EditorBrowsable(false)]` and possibly be a bit stronger and point them at `GetCheckpointAsync` in the `<remarks>` and potentially an `[Obsolete]` ?

The interplay between the pure `abstract` here and the `virtual` for `GetCheckpointAsync` may make this tricky.... it may be that non-browsable with a remark is the best we can do.... thoughts?

Reply:
`[EditorBrowsable(false)]` won't help, whoever inherits from `EventProcessor` would need to implement this method. I would also not rush to obsolete it because that would cause a compiler error for people with WarnAsError and we might realize we need `ListCheckpointsAsync` in some scenarios.

Reply:
Added a remark that overriding both is recommended.